pkg/services/dashboards/database/database.go

```diff
@@ -1043,6 +1043,40 @@ func (d *dashboardStore) CountDashboardsInFolders(
 	return count, err
 }
 
+// GetAllDashboardsUIDsInFolders returns all dashboards associated with the given folder list and orgID.
+// this will return soft deleted dashboards too (deleted IS NOT NULL)
+func (d *dashboardStore) GetAllDashboardsUIDsInFolders(
```

Comment:
This is a new function in the Dashboard Service since we need a list of dashboards UIDs by folder UID to delete associated public dashboards.

Reply:
Would it be possible instead to use `SearchDashboards` and pass in the list of folder uids & the org id?

Reply:
`SearchDashboards` is part of the `DashbaordService` and can't be imported on the folderService because of a circular dependency. I've used the `store.FindDashboards` instead which is what the `SearchDashboards` use under the hood.